tmk/README.md

```diff
@@ -77,12 +77,30 @@ Notes:
 ## Windows users
 
 Some basic modifications have been made to enable support for Windows. To compile, you will also need to install `make`, e.g., via [choco](https://community.chocolatey.org/packages/make) and [MinGW](https://osdn.net/projects/mingw/).
-  - MinGW needs to be added to the system path in order to compile and use the executables
+
+- MinGW needs to be added to the system path in order to compile and use the executables
+
+Known issues:
+
+- The tests will not run after `make` without more changes, but they can be run manually
+- Avoid BOM or CRLF line endings (e.g., if `haystack.txt` files are supplied)
+- Use of absolute file paths may need more work in some cases. Try relative paths, or copying executables (like `ffmpeg.exe`) to your local directory.
```

Comment:
Can you create issues for these?

Reply:
Sure. I combined them into a single Windows-based issue: https://github.com/facebook/ThreatExchange/issues/971